src/haxeLanguageServer/Context.hx

```diff
@@ -154,7 +154,7 @@ class Context {
         workspacePath = params.rootUri.toFsPath();
         capabilities = params.capabilities;
         var options = (params.initializationOptions : InitOptions);
-        if (options == null) {
+        if (options == null || Reflect.fields(options).length == 0 ) {
```

Comment:
Hm, curious.. @kLabz you didn't need anything like this, right?

Reply:
I did actually, I have the exact same fix atm